example-puzzles/example/200/answer.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+submittedAns=$1
+#This just outputs 't' plus whatever it is given.  MOTH wants to see an output of "true"
```

Comment:
I thought about return code.  The problem is that return code sometimes have actual meaning, depending on the OS, and no real standardization so I would be hesitant to use them.  IE if the code hits a certain type of exception, the language used or OS might auto return a particular value, and we do not know what that value would be.  It seems like standard in and out are highly consistent across platforms, which is why I prefer them.

Reply:
There are two levels of standards for return codes:

Level 1: 0 = success, anything else = failure
Level 2: defined in `/usr/include/stderr.h` [link](https://www.freebsd.org/cgi/man.cgi?query=sysexits&apropos=0&sektion=0&manpath=FreeBSD+4.3-RELEASE&format=html)

I propose we support level 1, where 0 means "you get points" and anything else means "you don't get points".

This is making me think, though, that it's going to be super helpful to percolate up to the user anything printed to stderr, as well as the return code, to make for easier debugging.

Reply:
I like the percolation idea, and can integrate that.  It could be super helpful in giving feedback on answers as well as debugging of course.  I am less enthusiastic about using return codes/exit status because I worry that certain compilers and programming languages might have weird default exit statuses for things like different exception types.  For instance, if an exception is thrown during execution because of something like parsing a malicious answer, the language/compiler could have default handling for that exception and return 0 as the exit status because it already handled the exception.  That would then be considered success.  Stdout requires much more deliberate and specified behavior.